sdk/desktopvirtualization/azure-resourcemanager-desktopvirtualization/CHANGELOG.md

```diff
@@ -1,14 +1,122 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2023-10-11)
```

Comment:
Release version is not correct, please hold on.

Reply:
The version number described in the branch `main` is `1.1.0-beta.1`.

Reply:
stable release version should be 1.1.0 in this case

Reply:
@haolingdong-msft @XiaofeiCao 
Is the release of issue#4592 suspended ?

Reply:
Yeah, we may need to wait until 3rd week to release lite.

Also, please set `autoVersioning: true` in release tool for non-first-GA SDKs(currently should all be the case).

Reply:
Ok，I will wait until week 3 to release the lite version again。